pbs/pbsrequest.go

```diff
@@ -127,6 +135,17 @@ func ParsePBSRequest(r *http.Request, cache cache.Cache) (*PBSRequest, error) {
 	if pbsReq.Device == nil {
 		pbsReq.Device = &openrtb.Device{}
 	}
+	if pbsReq.SDK == nil {
+		pbsReq.SDK = &SDK{}
+	}
+	if pbsReq.SDK.Version != "0.0.1" {
+		if pbsReq.PBSUser != nil {
```

Comment:
I don't think its necessary to do that at the moment. We currently don't forward the user object to all bidders in Prebid Server, so if we want to add that capability we should do it in a separate PR.